src/languages/c-like.js

```diff
@@ -225,10 +225,9 @@ export default function(hljs) {
       },
       {
         className: 'class',
-        beginKeywords: 'class struct', end: /[{;:]/,
+        beginKeywords: 'enum class struct', end: /[{;:<>=]/,
         contains: [
-          { beginKeywords: "final" },
-          {begin: /</, end: />/, contains: ['self']}, // skip generic stuff
+          { beginKeywords: "final class struct enum" },
```

Comment:
And I don’t think `enum` is required here.

Reply:
Yeah, but doesn't hurt either. :-)